avaframe/log2Report/generateCompareReport.py

```diff
@@ -0,0 +1,247 @@
+"""
+    Generate a markdown report for data provided in dictionary with format to compare two simulations
+
+    This file is part of Avaframe.
+"""
+
+# Load modules
+import os
+import glob
+import logging
+import numpy as np
+import shutil
+from avaframe.in3Utils import fileHandlerUtils as fU
+
+# create local logger
+# change log level in calling module to DEBUG to see log messages
+log = logging.getLogger(__name__)
+
+
+def fetchBenchParameters(avaDir):
+    """ Collect simulation parameter info from all standard tests """
+
+    # get name of avalanche
+    avaName = os.path.basename(avaDir)
+    avaDictName = avaName + 'Dict'
+
+    # set desired benchmark simulation info dictionary
+    if avaDictName == 'avaBowlDict':
+        avaDictName = {'simName': 'release1BL_null_dfa_0.155',
+                    'Simulation Parameters': {
+                        'type': 'list',
+                        'Release Area': 'releasehh1BL',
```

Comment:
```suggestion
                        'Release Area': 'release1BL',
```
i guess?

Reply:
yeah this was just for testing that the red highlighting is shown, but changed so that is correct now